spatial-tracking-explainer.md

```diff
@@ -148,20 +148,20 @@ function onSessionStarted(session) {
 ```
 
 ### Viewer reference spaces
-A _viewer_ reference space tracks the origin of the viewer. This type of reference space is primarily used for creating inline experiences with no tracking information. Instead, developers may use `XRReferenceSpace.originOffset` which is described in the [Application supplied transforms section](#application-supplied-transforms). An example usage of an _viewer_ reference space is a furniture viewer that will use [click-and-drag controls](#click-and-drag-view-controls) to rotate the furniture. It also supports cases where the developer wishes to avoid displaying any type of tracking consent prompt to the user prior while displaying inline content.
+A _viewer_ reference space's origin is always at the position and orientation of the viewer device. This type of reference space is primarily used for creating inline experiences with no tracking of the viewer relative to it's physical environment. Instead, developers may use `XRReferenceSpace.originOffset` which is described in the [Application supplied transforms section](#application-supplied-transforms). An example usage of an _viewer_ reference space is a furniture viewer that will use [click-and-drag controls](#click-and-drag-view-controls) to rotate the furniture. It also supports cases where the developer wishes to avoid displaying any type of tracking consent prompt to the user prior while displaying inline content.
```

Comment:
> A _viewer_ reference space's origin is always at the position and orientation of the viewer device.

Eh... now this sounds in conflict with the behavior of the originOffset.

Reply:
```suggestion
A _viewer_ reference space's native origin is always at the position and orientation of the viewer device. This type of reference space is primarily used for creating inline experiences with no tracking of the viewer relative to it's physical environment. Instead, developers may use `XRReferenceSpace.originOffset` which is described in the [Application supplied transforms section](#application-supplied-transforms). An example usage of an _viewer_ reference space is a furniture viewer that will use [click-and-drag controls](#click-and-drag-view-controls) to rotate the furniture. It also supports cases where the developer wishes to avoid displaying any type of tracking consent prompt to the user prior while displaying inline content.
```

Reply:
I guess the way to fix that would be to lift the concept of "native origin" into the explainer, but I'm not convinced that's necessary.

Reply:
I guess the explainer doesn't talk of originOffset at all. Hm. I'd say we should fix this separately, if at all, and for now just say "origin" to be consistent with the rest?

Reply:
That's fine, though we should call out that this will need to be clarified when we fix the issue @toji will file to address originOffset in the explainer. :)